rift/common/__init__.py

```diff
@@ -1,5 +1,11 @@
-#very, very magic code to cleanup the immutable schema to allow old code to modify rift packets
```

Comment:
I noticed you first deleted the magic code below and then you put it back in a later commit. But you forgot to put the comment back.

Reply:
oversight, will plug back in